src/aind_metadata_service/tars/client.py

```diff
@@ -79,6 +79,26 @@ def _get_prep_lot_response(
         response = requests.get(query, headers=headers)
         return response
 
+    def _get_molecules_response(
+        self, plasmid_name: str
+    ) -> requests.models.Response:
+        """
+        Retrieves molecules from TARS.
+        Parameters
+        ----------
+        plasmid_name: str
```

Comment:
I might be confused here, but are you defining a field called "plasmid_name" as the prep lot number? Let's not do that please. The plasmid is a defined thing. Plasmid -> virus -> viral prep lot.

Reply:
I'm guessing this is just a doc string copy/paste thing, right?

Reply:
Oh yes thank you for catching that, it was a copy/paste typo. I am using the plasmid name to fetch data from the molecular registry. I'll fix the docstring